Lib9c/Action/TransferAssets.cs

```diff
@@ -75,17 +76,21 @@ public override IAccountStateDelta Execute(IActionContext context)
                 return Recipients.Aggregate(state, (current, t) => current.MarkBalanceChanged(t.amount.Currency, new[] {Sender, t.recipient}));
             }
 
+            if (Recipients.Count > RecipientsCapacity)
+            {
+                throw new ArgumentOutOfRangeException($"{nameof(Recipients)} must be less than or equal {RecipientsCapacity}.");
+            }
             var addressesHex = GetSignerAndOtherAddressesHex(context, context.Signer);
             var started = DateTimeOffset.UtcNow;
-            Log.Debug("{AddressesHex}TransferAsset3 exec started", addressesHex);
+            Log.Debug("{AddressesHex}TransferAssets exec started", addressesHex);
```

Comment:
```suggestion
            Log.Debug("{AddressesHex} transfer_assets exec started", addressesHex);
```

it's trivial, but I think `action_type` can be more helpful.